pkg/http/api.go

```diff
@@ -108,6 +111,17 @@ func (a *api) constructStateEndpoints() []Endpoint {
 	}
 }
 
+func (a *api) constructTransactionEndpoints() []Endpoint {
```

Comment:
@amanbha I would think that a transactional set would just be an attribute in Options and not a new API.